test/unit/models/ep_postmaster/mailgun_post_test.rb

```diff
@@ -53,5 +53,27 @@ class MailgunPostTest < ActiveSupport::TestCase
       refute mailgun_post.undeliverable_email?
     end
 
+    should "use api_key when mailgun_api_key is not a lambda" do
+      api_key = "key-abc123"
+      EpPostmaster.configure { |config| config.mailgun_api_key= "#{api_key}" }
+      params = DummyParams.new(from: "email+somedomain.com@email.church360.org", to: "doesntexist@test.test", event: :bounced_email, mailgun_api_key: "#{api_key}").to_params
+      bounced_notification = MailgunPost.new(params)
+      assert_equal api_key, bounced_notification.api_key
+    end
+
+    should "use the api_key wnen the mailgun_api_key is a lamdaa" do
+      api_key = "key-pelham123"
+      EpPostmaster.configure { |config| config.mailgun_api_key= ->(message) { "#{api_key}" } }
```

Comment:
Do we anticipate the api key being a Lambda now?

Reply:
Yes. Because in members we have two possible sources of bounced emails.